src/backend/core/country/urls.py

```diff
@@ -12,4 +12,8 @@
     path("<str:code>/", views.CountryDetailView.as_view()),
     path("address/shipping/<int:id>/", views.ShippingAddressDetailView.as_view()),
     path("address/billing/<int:id>/", views.ShippingAddressDetailView.as_view()),
+    path("address/shipping/", views.ShippingInfoListView.as_view()),
```

Comment:
I'd vote for having these URLs prefixed by either `dashboard` or `storefront` to distinguish their usage, like in other views.
I suppose that all of these views are intended for dashboard, so I'd prefix them with `dashboard`

Reply:
Fixed.